noaa/gefs/forecast/reformat.py

```diff
@@ -36,8 +38,8 @@
 )
 
 _PROCESSING_CHUNK_DIMENSION = "init_time"
-_CRON_SCHEDULE = "0 7 * * *"  # At 7:00 UTC every day.
-
+_OPERATIONAL_CRON_SCHEDULE = "0 7 * * *"  # At 7:00 UTC every day.
+_VALIDATION_CRON_SCHEDULE = "0 10 * * *"  # At 10:00 UTC every day.
```

Comment:
```suggestion
_VALIDATION_CRON_SCHEDULE = "0 9 * * *"  # At 9:00 UTC every day.
```
They definitely finish in under two hours (i think its about 75 mins now)